clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/HttpElasticsearchDAO.scala

```diff
@@ -63,15 +64,15 @@ class HttpElasticsearchDAO private[dataaccess] (
     val responsePublisher = httpClient.publisher(searchDefinition)
     Source
       .fromPublisher(responsePublisher)
-      .map(_.to[Json].as[D](index.decoder).fold(throw _, identity))
+      .map(_.to[Json])
   }
 
   override def getMostRecentDocument(
     implicit index: ElasticsearchIndex[_]
   ): Future[Option[Json]] = {
     val searchDefinition = searchWithType(index.indexName / index.indexType)
       .size(1)
-      .sortByFieldDesc(ClioDocument.UpsertIdElasticSearchName)
+      .sortByFieldDesc(ElasticsearchUtil.toElasticsearchName(UpsertId.UpsertIdFieldName))
```

Comment:
Both `toElasticSearchName()` and `UpsertIdFieldName` are static, so the value of this function call won't change, so it seems like it should be done once as a property of `UpsertId`.

Reply:
In order to do that, I'd either have to put another magic string in `UpsertId` (not great but probably not problematic) or include `clio-dataaccess-model` in `clio-util` (probably not ok). Are you ok with the magic string?

Reply:
Not sure what you mean by magic string, my thinking was that we could add a `val` to `UpsertId` that contains the value of the expression `ElasticsearchUtil.toElasticsearchName(UpsertId.UpsertIdFieldName)`.  If that's not possible due to module dependencies you could make it a `val` in `HttpElasticsearchDAO` (which might be a better place for it anyway).

Reply:
I meant something like getting rid of `UpsertId.UpsertIdFieldName` entirely and creating a `val UpsertIdElasticsearchName = "upsert_id"` in `ElasticsearchIndex`.

Reply:
OK with me, I didn't realize we no longer had a use for `UpsertIdFieldName` other than for making the ES name.